### PR TITLE
[cesium] - code cleanup

### PR DESCRIPTION
**Pull request by LeonLiur**
# Fix Pull Request Template

## Key Information

- **Linear Issue**: [SY-948](https://linear.app/synnaxlabs/issue/SY-948/cesium-unit-test-review)

## Interface changes

Added a SetAuthority method to cesium/writer – setAuthority is currently in use by the console (aether), but is not tested by any client or by cesium.

## Description

Added test cases where lacking

## Basic Readiness Checklist

- [x] I have performed a self-review of my code.
- [x] I have added sufficient regression tests to cover the changes.

## Manual QA Additions

- [x] I have updated the [Release Candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template
  with necessary manual QA steps to test my change.

### CODE REVIEW

**Review by Lham42 — APPROVED** [review]

File: cesium/delete_test.go [hunk 1]
```diff
@@ -468,6 +468,14 @@ var _ = Describe("Delete", func() {
 						_, err = db.RetrieveChannel(ctx, index2)
 						Expect(err).To(BeNil())
 					})
+					It("Should error when there is an error in deleting index channels", func() {
```
Comment (emilbon99) [inline cesium/delete_test.go:471]: Things should error when there is an error ... hmmmmmmmmmmmmm

File: cesium/iterator_test.go [hunk 2]
```diff
@@ -95,6 +95,63 @@ var _ = Describe("Iterator Behavior", func() {
...
+						}))
+						Expect(i.SeekLast()).To(BeTrue())
+						Expect(i.Prev(5 * telem.Second)).To(BeTrue())
+						f := i.Value()
+						Expect(f.Get(data1Key)[0].Data).To(EqualUnmarshal([]uint16{22, 25}))
+						Expect(f.Get(data2Key)[0].Data).To(EqualUnmarshal([]uint16{11, 12, 13, 15}))
+						Expect(i.Close()).To(Succeed())
+					})
+
+					Specify("LE", func() {
```
Comment (emilbon99) [inline cesium/iterator_test.go:112]: Make it `SeekLE & Next` or something. Just calling it `LE` is confusing

File: cesium/iterator_test.go [hunk 3]
```diff
@@ -95,6 +95,63 @@ var _ = Describe("Iterator Behavior", func() {
...
+						}))
+						Expect(i.SeekLE(4 * telem.SecondTS)).To(BeTrue())
+						Expect(i.Next(6 * telem.Second)).To(BeTrue())
+						f := i.Value()
+						Expect(f.Get(data1Key)[0].Data).To(EqualUnmarshal([]uint16{14, 17, 19}))
+						Expect(f.Get(data2Key)[0].Data).To(EqualUnmarshal([]uint16{6, 8}))
+						Expect(i.Close()).To(Succeed())
+					})
+
+					Specify("GE", func() {
```
Comment (emilbon99) [inline cesium/iterator_test.go:125]: Same as above

**Review by emilbon99 — APPROVED** [review]
Approved after you look at my comments

**Pull request merged**